Floofbot/Modules/Help.cs

```diff
@@ -73,5 +75,74 @@ public async Task HelpCommand()
                 Trash = true
             });
         }
+
+        [Summary("Show help for a specific module")]
+        [Command("help")]
+        public async Task HelpCommand([Summary("module")]string requestedModule)
+        {
+            try
+            {
+                List<string> moduleNames = new List<string>();
+                List<CommandInfo> commands = _commands.Commands.ToList();
+                List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
+                List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
+
+                // put available module names into a list instead of their object
+                foreach (ModuleInfo module in _commands.Modules.ToList())
+                    moduleNames.Add(module.Name.ToLower());
+
+                if (!moduleNames.Contains(requestedModule.ToLower())) // no such cmd
+                {
+                    await Context.Channel.SendMessageAsync("Unable to find that command");
+                    return;
+                }
+
+                foreach (CommandInfo cmd in commands)
```

Comment:
This part can be refactored in a future PR -- we have some duplicated code going on.